### ISSUE TITLE
Default template displays constructor parameters when `@hideconstructor` is used

### ISSUE THREAD

**Issue by hegemonic** [jsdoc3/jsdoc#1397]
The default template shouldn't display the parameters for the following constructor, but it does:

```js
/**
 * @classdesc Toaster singleton.
 * @class
 * @param {string} foo - the foo
 * @hideconstructor
 */
function Toaster() {}
```

**Comment by hegemonic** [comment]
Fixed on master and releases/3.5. The fix will be included in JSDoc 3.5.2.

**Comment by schmidt-sebastian** [comment]
Just FYI, the constructor parameters are still visible in the heading for each class. Thanks!

**Comment by hegemonic** [comment]
@schmidt-sebastian So close and yet so far. 😭 I'll fix this in JSDoc 3.5.4.

**Comment by hegemonic** [comment]
Fixed on master. The fix will be included in JSDoc 3.5.4.
